src/runner.sh

```diff
@@ -28,12 +28,19 @@ function Runner::callTestFunctions() {
 function Runner::runTest() {
   local function_name="$1"
   local current_assertions_failed
+  local test_result_code=1
```

Comment:
What about using 0 as the default value, as 0 means success in bash?

Reply:
Yes, you are right 👍